cmd/mdmctl/apply.go

```diff
@@ -56,6 +60,7 @@ Apply a resource.
 Valid resource types:
 
   * blueprints
+  * dep-tokens
```

Comment:
I'd like to see the command itself be `dep`.

Imagine a syntax like:
```
mdmctl dep import-token /path/to/token-file.p7m
mdmctl dep export-token /path/to/stoken.json
mdmctl dep profiles (lists profiles)
mdmctl dep devices (list devices in DEP)
```

Thoughts? 
I think the `get/apply` commands are great but there's some value in separating DEP resources from mdm server ones. 

PS: I'm not opposed to the current interface either but wanted to bring it up for discussion.